### PR TITLE
bump openshift/api to prebase-1.18.beta2

### PR DESCRIPTION
**Pull request by soltysh**
/assign @sttts

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,12 +3,10 @@ module github.com/openshift/client-go
...
-	github.com/openshift/api v0.0.0-20200116145750-0e2ff1e215dd
+	github.com/openshift/api v0.0.0-20200311151921-fdf269f98861
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.18.0-beta.2
 	k8s.io/apimachinery v0.18.0-beta.2
 	k8s.io/client-go v0.18.0-beta.2
 	k8s.io/code-generator v0.18.0-beta.2
 )
-
-replace github.com/openshift/api => github.com/openshift/api v0.0.0-20200306192528-e5737622441f
```
Comment (sttts) [inline go.mod]: why is this gone?

File: go.mod [hunk 2]
```diff
@@ -3,12 +3,10 @@ module github.com/openshift/client-go
 go 1.13
 
 require (
-	github.com/openshift/api v0.0.0-20200116145750-0e2ff1e215dd
+	github.com/openshift/api v0.0.0-20200311151921-fdf269f98861
```
Comment (sttts) [inline go.mod]: hash looks good, corresponds to https://github.com/openshift/api/tree/prebase-1.18.beta2

File: go.mod (hunk 1 above)
Comment (soltysh) [reply go.mod]: Not needed, it pins the right version above.

File: go.mod (hunk 2 above)
Comment (soltysh) [reply go.mod]: Yup, the previous did not.

**Pull request merged**